### PR TITLE
SAK-42174 Switch, Gradebookng, Site, Reset-pass: implemented new banner classes

### PR DESCRIPTION
**Pull request by wyu12**
https://jira.sakaiproject.org/browse/SAK-42174

Replaced various banners (info, success, warn, error) in **gradebook**, **site**, **reset-pass**, with new banners created for SWITCH in SAK-41866. Also removed some unnecessary inline styles around/attached to these banners.

### CODE REVIEW

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.html [hunk 1]
```diff
@@ -4,7 +4,7 @@ <h2><wicket:message key="importExport.confirmation.heading" /></h2>
     <p><wicket:message key="importExport.confirmation.description" /></p>
 
     <div wicket:id="grades_update_container">
-    	<div class="messageInformation"><strong><wicket:message key="importExport.confirmation.update.heading" /></strong></div>
+    	<div class="sak-banner-info"><strong><wicket:message key="importExport.confirmation.update.heading" /></strong></div>
```
Comment (fostersdesign) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.html]: Do you think we still need these <strong> tags here?

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h2><wicket:message key="importExport.confirmation.heading" /></h2>
     </div>
 
     <div wicket:id="grades_create_container">
-   		<div class="messageInformation"><strong><wicket:message key="importExport.confirmation.create.heading" /></strong></div>
+   		<div class="sak-banner-info"><strong><wicket:message key="importExport.confirmation.create.heading" /></strong></div>
```
Comment (fostersdesign) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.html]: Do you think we still need these <strong> tags here?

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.html [hunk 3]
```diff
@@ -49,7 +49,7 @@ <h2><wicket:message key="importExport.confirmation.heading" /></h2>
     
     <!--  TODO collapse these three containers -->
     <div wicket:id="grades_modify_container">
-   		<div class="messageInformation"><strong><wicket:message key="importExport.confirmation.modify.heading" /></strong></div>
+   		<div class="sak-banner-info"><strong><wicket:message key="importExport.confirmation.modify.heading" /></strong></div>
```
Comment (fostersdesign) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.html]: Do you think we still need these <strong> tags here?

File: reset-pass/account-validator-tool/src/webapp/templates/transferMemberships.html [hunk 4]
```diff
@@ -40,7 +40,7 @@ <h3 rsf:id="welcome1">Welcome to Sakai!</h3>
 							<label><span rsf:id="msg=password">Password</span><span class="reqStar">*</span></label>
 							<input id="password" type="password" class="inputBox required" rsf:id="password" autocomplete="off" name="userpass" onfocus="VALIDATOR.checkTransferStatus();" onkeyup="VALIDATOR.checkTransferStatus();" oninput="VALIDATOR.checkTransferStatus();" required="required"/>
 						</div>
-						<p id="validate.loginexisting" rsf:id="validate.loginexisting" class="messageValidation">Discard the account...</p>
+						<p id="validate.loginexisting" rsf:id="validate.loginexisting" class="sak-banner-success">Discard the account...</p>
```
Comment (fostersdesign) [inline reset-pass/account-validator-tool/src/webapp/templates/transferMemberships.html]: Should this be a warning instead?

**Comment by wyu12** [comment]
removed `<strong>` tags for consistency and also updated the reset pass to sak-banner-warn. Could you give me an example of the success messages that you noticed?

**Pull request merged**